.gitignore

```diff
@@ -11,5 +11,6 @@ rustc
 __pycache__
 .idea/
 .vscode/
+.fleet/
```

Comment:
Personally, I feel like we shouldn't be ignoring every random editor's special files but should instead have users configure their personal gitignores to ignore their editor.

(note, this applies to more than just fleet)

Reply:
I second this, though not opposed to add it if it becomes popular.

See also: https://github.com/rust-lang/cargo/pull/7578

Reply:
I think that is a good solution! I'll close this PR for now then